domain/status/leadership_test.go

```diff
@@ -49,7 +49,7 @@ func (s *leadershipSuite) SetUpTest(c *tc.C) {
 	s.ModelSuite.SetUpTest(c)
 
 	modelUUID := uuid.MustNewUUID()
-	err := s.TxnRunner().StdTxn(c.Context(), func(ctx context.Context, tx *sql.Tx) error {
+	err := s.ModelSuite.TxnRunner().StdTxn(c.Context(), func(ctx context.Context, tx *sql.Tx) error {
```

Comment:
You don't have to do this. But just so you are aware when in tests there is a quicker way to do this. You can go something like.

`s.ModelSuite.DB().Exec`.

Reply:
Either approach is fine. Just an FYI for your toolkit.

Reply:
Sure & thanks tom for the info :D